releasetasks/test/test_releasetasks.py

```diff
@@ -521,3 +521,44 @@ def test_final_verify_task_definition(self):
             "queue:task-priority:high",
         ])
         self.assertTrue(expected_graph_scopes.issubset(graph["scopes"]))
+
+    def test_bouncer_submission_task_definition(self):
+        graph = make_task_graph(
+            version="42.0b2",
+            appVersion="42.0",
+            buildNumber=3,
+            source_enabled=False,
+            l10n_config={},
+            repo_path="releases/foo",
+            product="firefox",
+            revision="fedcba654321",
+            partial_updates={
+                "38.0": {
+                    "buildNumber": 1,
+                },
+                "37.0": {
+                    "buildNumber": 2,
+                },
+            },
+            branch="foo",
+            updates_enabled=False,
+            bouncer_enabled=True,
+            signing_class="release-signing",
+            verifyConfigs={}
+        )
+        self._do_common_assertions(graph)
+
+        task = get_task_by_name(graph, "release-foo_firefox_bncr_sub")
+
+        payload = task["task"]["payload"]
+
+        self.assertEqual(task["task"]["provisionerId"], "buildbot-bridge")
+        self.assertEqual(task["task"]["workerType"], "buildbot-bridge")
+        self.assertFalse("scopes" in task)
+        # XXX: Change the image name once it's in-tree.
+        self.assertTrue(payload["image"].startswith("rail/python-test-runner"))
```

Comment:
Can you also add some tests to make sure `payload.properties` has expected values. Something like
```python
self.assertEqual(payload["properties"]["partial_versions"] , "37.0, 38.0")
self.assertEqual(payload["properties"]["build_number"] , 3)
```
A hint: you may want to sort `partial_updates.iteritems()` to make it work properly. ;)